syntax = "proto3";


//声明grpc服务
service WrapKey{
     rpc Wrap (Cipher) returns (Cipher) {}
}


message Cipher{
    string transformation = 1;
    repeated bytes messages = 2;
    repeated Key wrappingKeys = 3;
}

message Key {
    string transformation = 1;
    int32 key_size = 2;
    bytes secret_key = 4;
}

